### PR TITLE
Remove Hubs from navigation

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
Mozilla Hubs is going to shut down in the next few months 😭 🌆. 
We're going to be removing the link to its site (as well as its easter egg) from our navigation.

Here is the blogpost explaining the shutdown timeline: https://hubs.mozilla.com/labs/sunset/

>[!WARNING]
> Merging this on February 29th

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/14232

## Testing
http://localhost:8000/en-US/

### CODE REVIEW

File: bedrock/base/templates/includes/protocol/navigation/menus/innovation.html [hunk 1]
```diff
@@ -19,15 +19,6 @@
       <button class="c-menu-button-close" type="button" aria-controls="c-menu-panel-innovation">{{ ftl('navigation-v2-close-innovation-menu') }}</button>
       <div class="c-menu-panel-content">
         <ul class="{{ row_count }}">
-          <li>
-            <section class="c-menu-item mzp-has-icon">
-              <a class="c-menu-item-link" href="https://hubs.mozilla.com/?{{ utm_params }}" data-link-text="Mozilla Hubs" data-link-type="nav" data-link-position="topnav" data-link-group="innovation">
-                <img loading="lazy" class="c-menu-item-icon" src="{{ static('img/nav/icons/icon-mozilla-hubs.svg') }}" width="32" height="32" alt="">
-                <h4 class="c-menu-item-title">{{ ftl('navigation-v2-mozilla-hubs') }}</h4>
-                <p class="c-menu-item-desc">{{ ftl('navigation-v2-gather-in-this-interactive-online', fallback='navigation-get-together') }}</p>
```
Comment (alexgibson) [inline bedrock/base/templates/includes/protocol/navigation/menus/innovation.html:27]: Can we more the string `navigation-get-together` as well?

File: media/css/protocol/components/_menu-item.scss [hunk 2]
```diff
@@ -156,25 +156,3 @@ $icon-size-md: 32px;
         display: block;
     }
 }
-
-// Easter egg
-@keyframes duckroll {
```
Comment (alexgibson) [inline media/css/protocol/components/_menu-item.scss:161]: Farewell 🦆 , hardly anyone knew you

**Review by alexgibson — APPROVED** [review]
r+wc

File: media/css/protocol/components/_menu-item.scss (hunk 2 above)
Comment (craigcook) [reply media/css/protocol/components/_menu-item.scss:161]: 😢

**Pull request merged**